megamek/src/megamek/common/BulldozerMovePath.java

```diff
@@ -78,7 +78,10 @@ public int getLevelingCost() {
     @Override
     public MovePath addStep(final MoveStepType type) {
         BulldozerMovePath mp = (BulldozerMovePath) super.addStep(type);
-
+        IHex hex = mp.getGame().getBoard().getHex(mp.getFinalCoords());
+        int hexWaterDepth = (hex != null) && hex.containsTerrain(Terrains.WATER) ?
```

Comment:
```suggestion
        int hexWaterDepth = ((hex != null) && hex.containsTerrain(Terrains.WATER)) ?
```

Reply:
This is a suggestion that I find makes it easier to read the line, but your mileage may vary.